src/sql-parser/src/parser.rs

```diff
@@ -3750,10 +3750,7 @@ impl<'a> Parser<'a> {
 
     fn parse_cluster_feature(&mut self) -> Result<ClusterFeature<Raw>, ParserError> {
         Ok(ClusterFeature {
-            name: self.parse_cluster_feature_name().map_err(|err| {
```

Comment:
This is simplifying some code that was merged with #25373 in a more complicated form than needed. Originally I was generating a generic error message in the synthesized `parse_~` method (`parse_cluster_feature_name` here) so I wanted to refine it at the call site. This is no longer needed in the version that got merged.